lib/puppet/http.rb

```diff
@@ -20,6 +20,7 @@ module HTTP
     require 'puppet/http/service/ca'
     require 'puppet/http/session'
     require 'puppet/http/resolver'
+    require 'puppet/http/resolver/serverlist'
```

Comment:
Since the constant is `Puppet::HTTP::Resolver::ServerList`, I suggest renaming this file to `server_list.rb`

Reply:
done